### DISCUSSION TITLE
Final project驗證遇到的問題

### DISCUSSION THREAD

**Discussion by KaiChen18** [bol-edu/HLS-SOC-Discussions#194] [Q&A]
<img width="787" alt="螢幕擷取畫面 2024-01-02 160813" src="https://github.com/bol-edu/HLS-SOC-Discussions/assets/145529354/f01fda4e-a3bb-4208-9069-e1cf377a64cf">

<img width="263" alt="螢幕擷取畫面 2024-01-02 161423" src="https://github.com/bol-edu/HLS-SOC-Discussions/assets/145529354/a11ae3e2-4f53-4e7c-92ad-743f4f4e9edf">

我們執行完後模擬的波型和結果應該是對的，但是上板子後無法回傳hello請問有人遇到類似的問題嗎?

**Comment by caillou100** [comment]
我沒有遇到這個問題,能否提供更多資訊呢

**Comment by Josephood7** [comment]
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/144367420/0d5b48f0-c62d-4368-8b6d-a3700c37559d)
因為python那邊好像會等收到hello才開始
看你們是要把它comment掉還是在la.c回傳一個hello
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/144367420/8b5d18f4-8d58-41ee-861a-db5baf7fa179)
